### PR TITLE
Add missing text domains and minor related adjustments

### PR DESCRIPTION
**Pull request by carolinan**
**Description**
Adds missing text domains.
Replace `esc_html__` with `esc_attr__` inside alt attribute.
Adds translator context to the "TBD" abbreviation.

### CODE REVIEW

**Comment by carolinan** [comment]
Do we also need to replace  ` ’` as in  `Oh hello. My name’s Angelo`, with `&#8217;` ?

File: inc/patterns/page-layout-image-and-text.php [hunk 1]
```diff
@@ -10,18 +10,18 @@
 					<h2 class="alignwide" style="font-size:clamp(4rem, 8vw, 7.5rem);font-weight:300;line-height:1.15">' . wp_kses_post( __( '<em>Watching Birds </em><br><em>in the Garden</em>', 'twentytwentytwo' ) ) . '</h2>
 					<!-- /wp:heading --></div>
 					<!-- /wp:group -->
-
```
Comment (carolinan) [inline inc/patterns/page-layout-image-and-text.php:13]: This spacing change is not meant to be here 🤔

File: inc/patterns/page-about-media-right.php [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
 					<div style="height:32px" aria-hidden="true" class="wp-block-spacer"></div>
 					<!-- /wp:spacer -->
 					<!-- wp:site-logo {"width":60} /-->
 
 					<!-- wp:group {"style":{"spacing":{"padding":{"right":"min(8rem, 5vw)","top":"min(20rem, 20vw)"}}}} -->
 					<div class="wp-block-group" style="padding-top:min(20rem, 20vw);padding-right:min(8rem, 5vw)"><!-- wp:heading {"style":{"typography":{"fontWeight":"300","lineHeight":"1.115","fontSize":"clamp(3rem, 6vw, 4.5rem)"}}} -->
-					<h2 style="font-size:clamp(3rem, 6vw, 4.5rem);font-weight:300;line-height:1.115"><em>Emery<br>Driscoll</em></h2>
+					<h2 style="font-size:clamp(3rem, 6vw, 4.5rem);font-weight:300;line-height:1.115"><em>' . esc_html__( 'Emery', 'twentytwentytwo' ) . '<br>' . esc_html__( 'Driscoll', 'twentytwentytwo' ) . '</em></h2>
```
Comment (MaggieCabrera) [inline inc/patterns/page-about-media-right.php]: Could this be `' . wp_kses_post( __( 'Emery<br>Driscoll', 'twentytwentytwo' ) ) . '` instead?

**Comment by jffng** [comment]
> Do we also need to replace ’ as in Oh hello. My name’s Angelo, with &#8217; ?

What would be the reason for doing this? The current way results in the correct right singular apostrophe.

**Review by jffng — COMMENTED** [review]
Pending @MaggieCabrera's comment, this LGTM 👍

File: inc/patterns/page-about-media-right.php (hunk 2 above)
Comment (carolinan) [inline inc/patterns/page-about-media-right.php]:
```suggestion
					<h2 style="font-size:clamp(3rem, 6vw, 4.5rem);font-weight:300;line-height:1.115"><em>' . wp_kses_post( __( 'Emery<br>Driscoll', 'twentytwentytwo' ) ) . '</em></h2>
```

**Comment by carolinan** [comment]
> > Do we also need to replace ’ as in Oh hello. My name’s Angelo, with ’ ?
> 
> What would be the reason for doing this? The current way results in the correct right singular apostrophe.

I remember needing to do that for TT1, but perhaps it was only inside HTML attributes.

**Review by jffng — APPROVED** [review]

**Pull request merged**